prac2-pages/anshumali_suri/lib/main.dart

```diff
@@ -0,0 +1,202 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+
+      home: Scaffold(
+
+        body: const MyStatefulWidget(),
+
+      ),
+    );
+  }
+}
+
+class MyStatefulWidget extends StatefulWidget {
+  const MyStatefulWidget({Key? key}) : super(key: key);
+
+  @override
+  State<MyStatefulWidget> createState() => _MyStatefulWidgetState();
+}
+
+class _MyStatefulWidgetState extends State<MyStatefulWidget> {
+  TextEditingController nameController = TextEditingController();
+  TextEditingController passwordController = TextEditingController();
```

Comment:
It is also a good practice to keep your instance variables as private. :thinking: 
This leads to a better implementation of the OOP principles.

Reply:
Resolved in commit [Update main.dart](https://github.com/RitwikSingh28/GitHubPrac/pull/15/commits/7d63d65c799f15f3d81c7dde0ead33dff3ae0ba8)